src/restaurant_inspections_api/handlers/businesses.clj

```diff
@@ -0,0 +1,55 @@
+(ns restaurant-inspections-api.handlers.businesses
```

Comment:
wow looking at this file I can see how much code we could reuse between inspections and businesses. you were right. But we'll take care of that in a different ticket. Maybe one after finishing this one heh.